src/libbson/CMakeLists.txt

```diff
@@ -390,22 +385,7 @@ if (ENABLE_APPLE_FRAMEWORK)
    )
 endif ()
 
-set (LIBBSON_LIBRARIES "")
-foreach (_lib ${BSON_LIBRARIES})
-   if (_lib MATCHES ".*/.*" OR _lib MATCHES "^-")
-      set (LIBBSON_LIBRARIES "${LIBBSON_LIBRARIES} ${_lib}")
-   else ()
-      set (LIBBSON_LIBRARIES "${LIBBSON_LIBRARIES} -l${_lib}")
-   endif ()
-endforeach ()
-# System dependencies don't match the above regexs, but also don't want the -l
-foreach (_lib ${BSON_SYSTEM_LIBRARIES})
-   set (LIBBSON_LIBRARIES "${LIBBSON_LIBRARIES} ${_lib}")
-endforeach ()
-
 include (CMakePackageConfigHelpers)
-set (INCLUDE_INSTALL_DIRS "${BSON_HEADER_INSTALL_DIR}")
-set (LIBRARY_INSTALL_DIRS ${CMAKE_INSTALL_LIBDIR})
```

Comment:
Only used by the old script generation during `configure_file`, so these aren't needed otherwise.